modules/stanford_news_views/stanford_news_views.views_default.inc

```diff
@@ -769,7 +769,9 @@ function stanford_news_views_views_default_views() {
   $handler->display->display_options['display_description'] = 'Homepage block of two items';
   $handler->display->display_options['defaults']['use_more'] = FALSE;
   $handler->display->display_options['use_more'] = TRUE;
-  $handler->display->display_options['link_url'] = 'news/recent-news';
+  $handler->display->display_options['defaults']['link_display'] = FALSE;
+  $handler->display->display_options['link_display'] = 'custom_url';
+  $handler->display->display_options['link_url'] = 'news/announcements';
```

Comment:
This change is fine but it will need a corresponding pull request for the menu items in the installation profile. Do we want the same sort of nesting as the current menu?